clients/fides-js/src/lib/tcf/i18n/tcf-i18n-utils.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * TCF-specific i18n utility functions. These are isolated to a separate file so
+ * that Rollup can easily remove this code from the base fides.js bundle without
+ * having to do any particularly difficult tree-shaking!
+ */
+import type { I18n, Locale } from "../../i18n";
+import { STATIC_MESSAGES_TCF } from "./locales";
```

Comment:
So basically, you've moved all the TCF-specific locales into the `tcf`-specific folder so that those don't get picked up and loaded into the message catalog by the non-TCF build. Then here, when we know we're working with the TCF-specific bundle, we load them in dynamically. Is that right?

Reply:
Yup, pretty much! The only difference is that it's not _really_ "dynamically" - it's statically true that this entire file (`tcf-i18n-utils`) is only reachable from the `fides-tcf.ts` entrypoint, so that makes it really obvious to rollup that it's not needed by the regular `fides.ts` entrypoint.